public-api/db-cdr/generate-cdr/run-measurement-queries.sh

```diff
@@ -655,344 +317,256 @@ round((case when m1.value_as_number < p10_value then p10_value
+      when m1.value_as_number between p10_value+5*((p90_value-p10_value)/11) and p10_value+6*((p90_value-p10_value)/11) then p10_value+6*((p90_value-p10_value)/11)
+      when m1.value_as_number between p10_value+6*((p90_value-p10_value)/11) and p10_value+7*((p90_value-p10_value)/11) then p10_value+7*((p90_value-p10_value)/11)
+      when m1.value_as_number between p10_value+7*((p90_value-p10_value)/11) and p10_value+8*((p90_value-p10_value)/11) then p10_value+8*((p90_value-p10_value)/11)
+      when m1.value_as_number between p10_value+8*((p90_value-p10_value)/11) and p10_value+9*((p90_value-p10_value)/11) then p10_value+9*((p90_value-p10_value)/11)
+      when m1.value_as_number between p10_value+9*((p90_value-p10_value)/11) and p10_value+10*((p90_value-p10_value)/11) then p10_value+10*((p90_value-p10_value)/11)
+      else p90_value
+     end),2)
+     end) as string) as stratum_4,
 count(distinct p1.person_id) as count_value,
 count(distinct p1.person_id) as source_count_value
-FROM \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m1
-join \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 on p1.person_id = m1.person_id
-join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c2 on c2.concept_id=m1.value_as_concept_id
+from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m1 join \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 on p1.person_id = m1.person_id
+join measurement_quartile_data on m1.measurement_concept_id=concept and unit='0'
 join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c1 on m1.measurement_concept_id=c1.concept_id
-where m1.value_as_concept_id != 0
-and m1.measurement_concept_id > 0
-and floor((extract(year from m1.measurement_date) - p1.year_of_birth)/10) >=3
-group by stratum_1,stratum_3,stratum_4,stratum_5
+where m1.measurement_concept_id > 0
+and m1.value_as_number is not null and p1.gender_concept_id=gender
+and m1.unit_concept_id = 0 and m1.unit_source_value is null
+group by stratum_1, stratum_2, stratum_3, stratum_4
 union all
-SELECT 0,1901 as analysis_id,
-cast(m1.measurement_source_concept_id as string) as stratum_1,'0' as stratum_2,CAST(floor((extract(year from m1.measurement_date) - p1.year_of_birth)/10) AS STRING) as stratum_3,
-c2.concept_name as stratum_4,
-cast(m1.value_as_concept_id as string) as stratum_5,
-count(distinct p1.person_id) as count_value,
-count(distinct p1.person_id) as source_count_value
-FROM \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m1
-join \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 on p1.person_id = m1.person_id
-join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c2 on c2.concept_id=m1.value_as_concept_id
-where m1.value_as_concept_id != 0
-and floor((extract(year from m1.measurement_date) - p1.year_of_birth)/10) >=3
-and m1.measurement_source_concept_id > 0 and
-m1.measurement_source_concept_id not in (select distinct measurement_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\`)
-group by stratum_1,stratum_3,stratum_4,stratum_5"
+select 0 as id, 1900 as analysis_id,
```

Comment:
Could you put a comment for this section? At a glance it looks identical to the above, what is different about this second query?

Reply:
Added.